storage/Filesystems/MountableImpl.cc

```diff
@@ -192,8 +207,10 @@ namespace storage
 	    createPath(real_mountpoint);
 	}
 
-	string cmd_line = MOUNTBIN " -t " + toString(get_mount_type()) + " " +
-	    quote(get_mount_name()) + " " + quote(real_mountpoint);
+	string cmd_line = MOUNTBIN " -t " + toString(get_mount_type());
+	if (!mount_opts.empty())
+	    cmd_line += " -o " + boost::join(mount_opts.get_opts(), ",");
```

Comment:
That's basically what mount_opts.format() would do. The only difference is that it would return "defaults" if empty, which you explicitly check one line before.